### DISCUSSION TITLE
Framework Laptop 13

### DISCUSSION THREAD

**Discussion by giscus** [framewiki/framewiki.net#22] [Talk]
# Framework Laptop 13

The unofficial community-driven Framework wiki

http://framewiki.net/framework-laptop-13

**Comment by Morpheus636** [comment]
The table includes factual inaccuracies. 13th gen and 7040 series both shipped with 55Wh batteries on the base config. 61Wh was only on the i7 and R7 configs.

**Comment by Morpheus636** [comment]
Links to the specs tab for 11th and 12th gen intel models are broken. Seems like we will need to replace with wayback links.

**Reply by Morpheus636** [reply]
Resolved.

**Comment by Morpheus636** [comment]
Link to  https://guides.frame.work/Guide/Framework+Laptop+13+DIY+Edition+Quick+Start+Guide is broken.

**Reply by Morpheus636** [reply]
Resolved.
